### DISCUSSION TITLE
Documentation and versions

### DISCUSSION THREAD

**Discussion by IgorKochubey** [ClickHouse/ClickHouse#65272] [Ideas]
Hello everyone,
first of all thanks for your project and documentation)

I have a idea about versions for documentation like https://www.postgresql.org/docs/9.5/index.html
I think it will be better to know which version contains new features, for example, https://clickhouse.com/docs/en/sql-reference/statements/delete is a good article, but not at all have the latest version.

 
![image](https://github.com/ClickHouse/ClickHouse/assets/57898267/d60b73ba-c4be-4903-bac7-58f960b70acc)

**Comment by UnamedRus** [comment]
> I have a idea about versions for documentation like https://www.postgresql.org/docs/9.5/index.html

It was discussed few times, https://github.com/ClickHouse/ClickHouse/issues/40304
